model/simulationtests/airterminal_inletsidemixer.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require 'openstudio'
+require 'lib/baseline_model'
+
+m = BaselineModel.new
+
+# make a 1 story, 100m X 50m, 1 zone core building
+m.add_geometry({ 'length' => 100,
+                 'width' => 50,
+                 'num_floors' => 1,
+                 'floor_to_floor_height' => 3,
+                 'plenum_height' => 0,
+                 'perimeter_zone_depth' => 0 })
+
+# add windows at a 40% window-to-wall ratio
+m.add_windows({ 'wwr' => 0.4,
+                'offset' => 1,
+                'application_type' => 'Above Floor' })
+
+# Add ASHRAE System type 07, VAV w/ Reheat, this creates a ChW, a HW loop and a
+# Condenser Loop
+m.add_hvac({ 'ashrae_sys_num' => '07' })
```

Comment:
Starting point. I'll swap the ATU with the ATU InletSideMixer + a Four Pipe FCU (so I also need HW/ChW loops)